README.md

```diff
@@ -111,13 +111,30 @@ module.exports = {
         maxSuggestions: 10, // Default: 5
         hotKeys: [], // Default: ['s', '/']
         cropLength: 50, // Default: 30
-        layout: 'simple' // Default: "columns"
+        layout: 'simple', // Default: "columns"
+        enableDarkMode: true // Default: false
       }
     ]
   ]
 }
 ```
 
+#### Dark mode
+
+You can enable dark mode by adding `enableDarkMode: true` to your configuration file.
+
+To override the default theme of the searchbar, you can edit your `.vuepress/styles/palette.styl` file.
```

Comment:
```suggestion
To override the default theme of the search bar, you can edit your `.vuepress/styles/palette.styl` file.<br>
```

Reply:
It's not convenient AT ALL, but markdown file does not consider your newline in the same paragraph. You have to put a `<br>` (or if you build markdown files, like in the documentation, download a plugin haha)